django-backend/fecfiler/contacts/views.py

```diff
@@ -40,7 +40,19 @@
 def validate_and_sanitize_candidate(candidate_id):
     if candidate_id is None:
         raise AssertionError("No Candidate ID provided")
-    return candidate_id
+    if re.match(r"^[a-zA-Z0-9]+$", candidate_id):
+        return candidate_id
+    else:
+        raise AssertionError("Candidate ID provided invalid")
+
+
+def validate_and_sanitize_committee(committee_id):
+    if committee_id is None:
+        raise AssertionError("No Committee ID provided")
+    if re.match(r"^[a-zA-Z0-9]+$", committee_id):
```

Comment:
can you update both of these to also be 9 characters

Reply:
Ok got it updated so it'll only accept if user puts in exactly 9 characters for those.